modules/data-streams/src/javaRestTest/java/org/elasticsearch/datastreams/logsdb/LogsIndexModeDisabledRestTestIT.java

```diff
@@ -54,4 +70,31 @@ public void testLogsSettingsIndexModeDisabled() throws IOException {
         assertThat(indexMode, Matchers.not(equalTo(IndexMode.LOGSDB.getName())));
     }
 
+    public void testTogglingLogsdb() throws IOException {
```

Comment:
Maybe move these tests to logsdb module?

Reply:
Apologies  for the confusion. I was just referring to the new tests that were added.

Reply:
To be more specific I mean moving `testTogglingLogsdb()`, `testEnablingLogsdb()`, `testTogglingLogsdb()` and `testLogsAtSettingWithTimeSeriesOverride()` to `LogsdbRestIT` in the logsdb module.

Reply:
Those methods use stuff that is in base class `LogsIndexModeRestTestIT`. So I either move all clasees ot I need to duplicate code creating another base class there or methods used by the test.

Reply:
Ok, in that case let's leave the tests where they are. In a follow up change we can move all logsdb integration tests to the logsdb module.